### PR TITLE
issue #49 resolved session section changed

### PR DESCRIPTION
**Pull request by aayusharora**

### CODE REVIEW

File: app/components/sessions/SessionsController.js [hunk 1]
```diff
@@ -1,107 +1,139 @@
...
-        sdc.duration = function(session) {
-            var start = DateUtils.getHourMin(session.begin);
-            var end = DateUtils.getHourMin(session.end);
+                    });
+                };/*Display only when id match */
+                sc.display=function(session,event){
+                   
+                    
+
+                   for(i=0;i<sc.Days.length;i++){
```
Comment (SaptakS) [inline app/components/sessions/SessionsController.js]: Please define 'i'. Though it will work nonetheless, but its good to define 'i'.

**Comment by aayusharora** [comment]
@championswimmer , I have to create this again , (deleted the branch mistake) , is it required ?

File: app/components/speakers/SpeakersController.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ speakersModule.controller('SpeakersController',
             ApiJsonFactory.getJson('speakers')
                 .then(function (response) {
                     sc.Speakers = response.data.speakers;
+                    console.log(sc.Speakers);
```
Comment (juslee) [inline app/components/speakers/SpeakersController.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535186/issues/source?bid=3233857&fileBranchId=3233878#l33)

File: app/components/speakers/SpeakersController.js [hunk 3]
```diff
@@ -43,6 +44,7 @@ speakersModule.controller('SpeakersController',
             ApiJsonFactory.getJson('sessions')
                 .then(function (response) {
                     sc.Sessions = response.data.sessions;
+                    console.log("hello");
```
Comment (juslee) [inline app/components/speakers/SpeakersController.js]: Codacy Issue found: [Unexpected console statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535186/issues/source?bid=3233857&fileBranchId=3233878#l47)

File: app/components/sessions/SessionsController.js [hunk 4]
```diff
@@ -1,107 +1,119 @@
...
+                            singleSession: session
+                        };
+                        $mdDialog.show({
+                            controller: 'SessionDialogController',
+                            templateUrl: 'app/components/sessions/sessiondialog.html',
+                            parent: angular.element(document.body),
+                            targetEvent: event,
+                                      });
+                    };/*Display only when id match */
+                    sc.display=function(session,event){
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l86)

File: app/components/sessions/SessionsController.js [hunk 5]
```diff
@@ -1,107 +1,119 @@
...
+                                 for(var i=0;i<sc.Days.length;i++){
+                                 for(var x in sc.Days[i].sessions){
+                                   if(sc.Days[i].sessions[x].id==session){
+                                   if( document.getElementById(session).style.display =="none"){
+                                   var n=document.getElementById(session);
+                                  n.style.display = "block";
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: ["n" used outside of binding context.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l92)

File: app/components/sessions/SessionsController.js [hunk 6]
```diff
@@ -1,107 +1,119 @@
...
+                            templateUrl: 'app/components/sessions/sessiondialog.html',
+                            parent: angular.element(document.body),
+                            targetEvent: event,
+                                      });
+                    };/*Display only when id match */
+                    sc.display=function(session,event){
+                                 for(var i=0;i<sc.Days.length;i++){
+                                 for(var x in sc.Days[i].sessions){
+                                   if(sc.Days[i].sessions[x].id==session){
+                                   if( document.getElementById(session).style.display =="none"){
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l90)

File: app/components/sessions/SessionsController.js [hunk 7]
```diff
@@ -1,107 +1,119 @@
...
+                            controller: 'SessionDialogController',
+                            templateUrl: 'app/components/sessions/sessiondialog.html',
+                            parent: angular.element(document.body),
+                            targetEvent: event,
+                                      });
+                    };/*Display only when id match */
+                    sc.display=function(session,event){
+                                 for(var i=0;i<sc.Days.length;i++){
+                                 for(var x in sc.Days[i].sessions){
+                                   if(sc.Days[i].sessions[x].id==session){
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l89)

File: app/components/sessions/SessionsController.js [hunk 8]
```diff
@@ -1,107 +1,119 @@
...
+                                   if( document.getElementById(session).style.display =="none"){
+                                   var n=document.getElementById(session);
+                                  n.style.display = "block";
+                                  event=true;
+                                              }
+                                            else{
+                                                 var n=document.getElementById(session);
+                                                  n.style.display = "none";
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: ["n" used outside of binding context.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l97)

File: app/components/sessions/SessionsController.js (hunk 4 above)
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: ["event" is defined but never used](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l86)

File: app/components/sessions/SessionsController.js [hunk 9]
```diff
@@ -1,107 +1,119 @@
...
+                                            }      
+                                       }
+                                    }
+                                }
                             }
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: [Missing semicolon.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l102)

File: app/components/sessions/SessionsController.js [hunk 10]
```diff
@@ -1,107 +1,119 @@
...
+                            parent: angular.element(document.body),
+                            targetEvent: event,
+                                      });
+                    };/*Display only when id match */
+                    sc.display=function(session,event){
+                                 for(var i=0;i<sc.Days.length;i++){
+                                 for(var x in sc.Days[i].sessions){
+                                   if(sc.Days[i].sessions[x].id==session){
+                                   if( document.getElementById(session).style.display =="none"){
+                                   var n=document.getElementById(session);
```
Comment (juslee) [inline app/components/sessions/SessionsController.js]: Codacy Issue found: ["n" used outside of binding context.](https://www.codacy.com/app/triplez/open-event-webapp/file/2406535184/issues/source?bid=3233857&fileBranchId=3233878#l91)

**Pull request closed without merging**